asserts/asserts.go

```diff
@@ -218,11 +220,173 @@ func Decode(serializedAssertion []byte) (Assertion, error) {
 		return nil, fmt.Errorf("parsing assertion headers: %v", err)
 	}
 
-	if len(signature) == 0 {
-		return nil, fmt.Errorf("empty assertion signature")
+	return Assemble(headers, body, content, signature)
+}
+
+// Maximum assertion component sizes.
+const (
+	MaxBodySize      = 2 * 1024 * 1024
+	MaxHeadSize      = 128 * 1024
```

Comment:
Can we call this MaxHeaderSize?  I think that's the traditional term used for that, and the one we use even in the code below.